tests/Doctrine/Tests/ORM/Functional/SchemaTool/MySqlSchemaToolTest.php

```diff
@@ -89,6 +105,89 @@ public function testGetCreateSchemaSql4()
         $this->assertEquals(0, count($sql));
     }
 
+    public function testUpdateSchemaSql()
+    {
+        $classes = [
+            $this->_em->getClassMetadata(MyEntityToRemove::class),
+        ];
+        $tool    = new SchemaTool($this->_em);
+        $sqls    = $tool->getUpdateSchemaSql($classes);
+        $sqls    = $this->filterSqls($sqls, ['entity_to_remove']);
+        $this->assertCount(1, $sqls);
```

Comment:
`self::assertCount()` since the method is static (other places should be fixed as well)